build/generate-style-spec.ts

```diff
@@ -317,9 +329,9 @@ ${objectDeclaration('LightSpecification', spec.light)}
 
 ${objectDeclaration('SkySpecification', spec.sky)}
 
-${objectDeclaration('TerrainSpecification', spec.terrain)}
+${objectDeclaration('ProjectionConfigSpecification', spec.projectionConfig)}
```

Comment:
Not sure I like it, but I don't think I have a better idea how to solve it.
I see that color is using ColorSpecification, so ProjectionSpecification is save to the spec of the projection object used inside the root projection definition.
the only idea that comes to my mind is `RootProjection` but that's not good...
Feel free to resolve this comment if you can't this of a better name for this type... I can't think of one right now...